source/Calamari.Common/Features/Discovery/KubernetesCluster.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace Calamari.Common.Features.Discovery
+{
+    public class KubernetesCluster
+    {
+        public KubernetesCluster(string name, string resourceGroupName, string accountId, TargetTags tags)
+        {
+            Name = name;
+            ResourceGroupName = resourceGroupName;
+            AccountId = accountId;
+            Tags = tags;
+        }
+
+        public string Name { get; }
+        
+        public string ResourceGroupName { get; }
```

Comment:
As these are Azure properties, we should revisit this when we implement EKS support

Reply:
Nothing to do now though.

Reply:
Yep ok good point 👍 we can do that in the next PR.